tests/integration/block_create_test.go

```diff
@@ -56,7 +56,7 @@ func (s *BlockCreateSuite) SetupSuite() {
 	s.namespace = "block-k8s-ns"
 	mons := 1
 	rbdMirrorWorkers := 1
-	s.op, s.kh = StartTestCluster(s.T, s.namespace, "bluestore", false, false, mons, rbdMirrorWorkers, installer.VersionMaster, installer.MimicVersion)
+	s.op, s.kh = StartTestCluster(s.T, s.namespace, "bluestore", true, false, false, mons, rbdMirrorWorkers, installer.VersionMaster, installer.MimicVersion)
```

Comment:
If we're going to have a parameter for enabling CSI, at least one of the tests should pass false.